test/python/utils.py

```diff
@@ -0,0 +1,121 @@
+import os
```

Comment:
I think the testing methods should be in the parflow package.

maybe

parflow.tools.compare

Putting the TCL utils in the test directory is a bit ugly when we started to add multiple test subdirectories.    People started to copy the file :(

Reply:
I agree - I'll move it. I was trying to copy the tcl structure but it fits better out of the test directory.